### DISCUSSION TITLE
Github education pack

### DISCUSSION THREAD

**Discussion by Farhan2724** [community/community#146840] [GitHub Education]
### Hi everyone,

Hi all,

I applied to github education pack a few ago, I'm kind of growing anxious because i have an upcoming project due midnight. 
Pls approve as soon as possible.

**Comment by cagansen-git** [comment]
my profile was accepted like 3-5 days ago too and my student pack is still waiting.

**Comment by davevad93** [comment]
Hi @Farhan2724 ,

GitHub no longer provide manual reviews; the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Farhan2724 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
